src/usbd.rs

```diff
@@ -30,8 +31,8 @@ pub struct InterruptHandler<T: Instance> {
 impl<T: Instance> interrupt::typelevel::Handler<T::Interrupt> for InterruptHandler<T> {
     unsafe fn on_interrupt() {
         let regs = T::regs();
-        let x = regs.istr().read().0;
-        crate::println!("USB IRQ: {:08x}", x);
+        // let x = regs.istr().read().0;
+        // crate::println!("USB IRQ: {:08x}", x);
```

Comment:
Note: These could simply be turned into `debug!`/`info!` / etc. calls, since without the `defmt` feature, those are simply no-ops. This will allow people with debugging needs to simply re-enable these printouts without needing to mess with the HAL code.